docsrc/index.rst

```diff
@@ -44,13 +44,10 @@ User Guide
    :hidden:
 
    user_guide/torch_tensorrt_explained
-   user_guide/getting_started
    user_guide/dynamic_shapes
    user_guide/saving_models
    user_guide/runtime
    user_guide/using_dla
-   tutorials/_rendered_examples/dynamo/torch_compile_advanced_usage
```

Comment:
why is this getting deleted?

Reply:
I thought it was already listed under Tutorials and I deemed it a duplicate.

Reply:
I'm not sure if you wanted to list the same page twice under different sections. But it seems unusual to me that a reader read the same page twice because it was indexed two times.

Reply:
Ah, I think we actually want to get rid of the later one if anything, seems like that it should be in the user guide as it is part of the primary workflow